packages/wrangler/index.js

```diff
@@ -0,0 +1,6 @@
+import { foo } from "./another";
```

Comment:
No idea how this file got in there. I'll remove it now.

Reply:
See https://github.com/cloudflare/workers-sdk/pull/6547